### PR TITLE
Overview 2

### PR DESCRIPTION
**Pull request by MennoWellner**

### CODE REVIEW

**Comment by MennoWellner** [comment]
PR made into a draft, because a few tasks are still open

File: src/app/components/main/header/header.component.html [hunk 1]
```diff
@@ -1,6 +1,4 @@
-<!--
- Header - navbar
- -->
+<!--Todo - Add an entry to the `My events` dropdown to be able to navigate to the overview 2. -->
```
Comment (NoNamer777) [inline src/app/components/main/header/header.component.html:1]: Technically you could remove this, because you've already done below what's asked.

Comment (NoNamer777) [reply src/app/components/main/header/header.component.html:1]: We'll get into routing at a later stage in the project, don't worry about that for now.

File: src/app/components/events/overview2/overview2.component.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            </div>
+        </div>
+        <div class="col">
+            <p>Select an event in the panel on the left.</p>
+        </div>
+        <div class="col"></div>
+    </div>
+</div>
+
+<!--TODO - As suggested in the task, use the ngif Angular syntax to show the detail2 template or an alternative message. -->
```
Comment (NoNamer777) [inline src/app/components/events/overview2/overview2.component.html:33]: You should be able to figure out how to implement this with the resources described here:
https://angular.io/guide/built-in-directives#adding-or-removing-an-element-with-ngif

File: src/app/components/main/header/header.component.html (hunk 1 above)
Comment (MennoWellner) [reply src/app/components/main/header/header.component.html:1]: Alrighty

File: src/app/components/events/overview1/overview1.component.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ export class Overview1Component {
         let event: AEvent;
 
         for (let i = 0; i < RANDOM_GENERATED_EVENTS; i++) {
-            event = new AEvent();
+            event = new AEvent(i + 1);
 
             event.title = `The Fantastic event-${i}`;
```
Comment (NoNamer777) [inline src/app/components/events/overview1/overview1.component.ts:26]: This title is also de-synchronized..

Comment (MennoWellner) [reply src/app/components/events/overview1/overview1.component.ts:26]: Yes as intended. But I've added the '+ 1' to the title now as well :)

**Review by NoNamer777 — APPROVED** [review]
LGTM

We can have a look at testing in another PR.

**Pull request merged**